app/javascript/components/mentoring/mentor_conversations_list.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import dayjs from 'dayjs'
+import RelativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(RelativeTime)
+
+function MentorConversationListRow({
+  trackIconUrl,
+  menteeAvatarUrl,
+  menteeHandle,
+  exerciseTitle,
+  isStarred,
+  haveMentoredPreviously,
+  isNewIteration,
+  postsCount,
+  updatedAt,
+  url,
+}) {
+  return (
+    <tr>
+      <td>
+        <img style={{ width: 100 }} src={trackIconUrl} />
+      </td>
+      <td>
+        <img style={{ width: 100 }} src={menteeAvatarUrl} />
+      </td>
+      <td>{menteeHandle}</td>
+      <td>{exerciseTitle}</td>
+      <td>{isStarred.toString()}</td>
+      <td>{haveMentoredPreviously.toString()}</td>
+      <td>{isNewIteration.toString()}</td>
+      <td>{postsCount}</td>
+      <td>{dayjs(updatedAt).from(dayjs())}</td>
```

Comment:
Instead you probably want to use `.fromNow()`

Reply:
Oh, yes. Thanks!